<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>affecterpro</class>
 <widget class="QDialog" name="affecterpro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/backgrouuuundd.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_afficherliste">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>380</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher la liste des vols</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>731</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_affecter">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>620</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Affecter un vol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_affecter">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>430</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1036</x>
     <y>430</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrer ID_vol</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>490</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrez id promo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_idpromo">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>490</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
